df_py/challenge/test/test_challenge_calc_rewards.py

```diff
@@ -12,7 +12,7 @@ def test_calc_challenge_rewards():
         "0xfrom3",
     ]
 
-    rewards = calc_challenge_rewards(from_addrs)
+    rewards = calc_challenge_rewards(from_addrs, datetime(2023, 10, 15))
```

Comment:
This change is in conflict with [PR #758](https://github.com/oceanprotocol/df-py/pull/758). PR 578 is dedicated to this fix, and points to the issue it's solving. Therefore I recommend undoing the change in _this_ PR.